cruditor/views.py

```diff
@@ -258,11 +260,29 @@ def perform_delete(self, form):
         """
         self.object.delete()
 
+    def format_linked_objects(self, objects):
+        """
+        Generate a list of strings describing the objects which have a protected
+        relation to the item to delete.
+        """
+        return [
+            ugettext('{model}: {object}').format(
+                model=capfirst(obj.__class__._meta.verbose_name), object=obj
```

Comment:
Do we need capfirst here? I thought verbose_name is already capitalized if not given explicitly.

Reply:
I took inspiration in fact from the admin of Django and from the contrib module `modeladmin` of Wagtail. They both capitalize the verbose name of the model - e.g. see [here](https://github.com/django/django/blob/2.2/django/contrib/admin/utils.py#L142) for Django - and I think it's because is generally in lowercase as the [documentation suggest](https://docs.djangoproject.com/en/2.2/ref/models/options/#verbose-name).

Reply:
Alright, thanks for the explanation!